repos/fdbt-site/src/utils/sessions.ts

```diff
@@ -148,6 +149,7 @@ import {
     SalesOfferPackage,
     AdditionalOperator,
     CapExpiry,
```

Comment:
/selectCapValidity page header and sub header needs changing from

When does the product expire?
We need to know the time that this product would be valid until

to

When does the cap expire?
We need to know the time that this cap would be valid until